patches/tModLoader/Terraria/Main.TML.cs

```diff
@@ -195,7 +195,7 @@ public partial class Main
 
 			base.Content = new TMLContentManager(Content.ServiceProvider, vanillaContentFolder, AlternateContentManager);
 		}
-		
+
```

Comment:
Please check your patches and try and avoid adjusting formatting in files unrelated to your PR